pootle/apps/pootle_data/store_data.py

```diff
@@ -27,7 +27,6 @@ def object_stats(self):
         stats = {
             v: getattr(self.context.data, k)
             for k, v in self.stats_mapping.items()}
-        stats["is_dirty"] = False
```

Comment:
`is_dirty` appears in tests, not strictly related to this. Maybe other appearances of this can be dropped as well.